testing/containers/ceph/Dockerfile

```diff
@@ -16,7 +16,8 @@ RUN true && \
     yum install -y \
     git wget curl make \
     /usr/bin/cc /usr/bin/c++ \
-    "libcephfs-devel-${cv}" "librados-devel-${cv}" "librbd-devel-${cv}" && \
+    "libcephfs-devel-${cv}" "librados-devel-${cv}" "librbd-devel-${cv}" \
+    gdb libcephfs2-debuginfo librados2-debuginfo librbd1-debuginfo && \
```

Comment:
I see a **librados-devel-debuginfo** package too. But if we could get a complete backtrace without it I am all good. May be that's some kind of packaging error.

Reply:
It contains `/usr/lib/debug/usr/bin/librados-config-16.2.14-0.el8.x86_64.debug` and I can't find any reference to librados-config in our code, so I think we can skip it.